doc/Tex/RAM_SCB_intro.tex

```diff
@@ -90,9 +43,9 @@ \section{TERMS OF USE \& DISTRIBUTION POLICY}
     \hline
     Welling et al. [2011] & First full description of one-way coupling with SWMF \\
     \hline
-    Welling et al. [2015] & Description of two-way coupling of RAM-SCB
-  with SWMF \\
+    Welling et al. [2015] & Description of two-way coupling of RAM-SCB with SWMF \\
     \hline
+    Engel et al. [2019] & Improved simulations of the inner magnetosphere during high geomagnetic activity with the RAM‐SCB model \\

```

Comment:
As above, I'd think about revising the list of what counts as "physics" versus "development milestones".